packages/frontend-2/nuxt.config.ts

```diff
@@ -186,23 +186,30 @@ export default defineNuxtConfig({
     },
     '/profile': {
       redirect: {
-        to: '/?settings=user/profile',
+        to: '/?settings/user/profile',
         statusCode: 301
       }
     },
-
-    // Redirect settings 'route' to homepage with added query
-    '/settings': { redirect: '/?settings=user/profile' },
-    '/settings/user/profile': { redirect: '/?settings=user/profile' },
-    '/settings/user/notifications': { redirect: '/?settings=user/notifications' },
-    '/settings/user/developer-settings': {
-      redirect: '/?settings=user/developer-settings'
+    '/settings/server/active-users': {
+      redirect: {
+        to: '/settings/server/members',
+        statusCode: 301
+      }
     },
-    '/settings/server/general': { redirect: '/?settings=server/general' },
-    '/settings/server/projects': { redirect: '/?settings=server/projects' },
-    '/settings/server/active-users': { redirect: '/?settings=server/active-users' },
     '/settings/server/pending-invitations': {
-      redirect: '/?settings=server/pending-invitations'
+      redirect: {
+        to: '/settings/server/members',
+        statusCode: 301
+      }
+    },
+    '/settings/**': {
+      appMiddleware: ['auth', 'settings']
+    },
+    '/settings/server/*': {
+      appMiddleware: ['auth', 'settings', 'admin']
```

Comment:
hoping these don't get added & invoked twice (cause of the /settings/** rule above)